src/frontend/src/Components/map/layers/regionalWeatherLayer.js

```diff
@@ -49,6 +49,8 @@ export function getRegionalWeatherLayer(weatherData, projectionCode, mapContext,
               updateReferenceFeature(olFeatureForMap);
             }
           }
+
+          console.log('Weather data:', weather.forecast_group);
```

Comment:
Does this console log need to stay for any reason?

Reply:
removed in latest commit